portfolio/src/main/webapp/script.js

```diff
@@ -144,6 +144,8 @@ fetch('./config.json')
       var script = document.createElement('script');
```

Comment:
if you can, try avoid using var as per [this](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)

Reply:
Thanks for pointing this out. Now replaced all with const